application/controllers/register.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require "./vendor/autoload.php";
+
+$validate = new Validation();
+$database = new Database();
+$pathOfProfilePhoto = "";
+
+if (isset($_POST['register'])) {
+  if (!$validate->isValidName($_POST['firstName'], $_POST['lastName'])) {
+    $GLOBALS["nameErrorMessage"] = $validate->nameError;
+  }
+  if (!$validate->isValidEmailAddress($_POST['email'])) {
+    $GLOBALS["emailIdErrorMessage"] = $validate->emailError;
+  }
+  // mail box layer API is not working so i comment it out.
+  // if (!$validate->isEmailAddressExits($_POST['email'])) {
+  //   $GLOBALS["emailExitsErrorMessage"] = $validate->emailExitsError;
+  // }
+  if ($validate->isValidProfilePhoto($_FILES['profileImage']['name'], $_FILES['profileImage']['type'], $_FILES['profileImage']['size'])) {
+    $pathOfProfilePhoto = "public/assets/image/profilePhoto/" . $_FILES['profileImage']['name'];
+    //If uploaded image is valid then send the image upload_image folder.
+    move_uploaded_file($_FILES['profileImage']['tmp_name'], $pathOfProfilePhoto);
+  }
+  else {
+    $GLOBALS['profilePhotoErrorMessage'] = $validate->uploadedFileError;
+  }
+  if (!$validate->isValidPassword($_POST['password'])) {
+    $GLOBALS["passwordErrorMessage"] = $validate->passwordError;
+  }
+  if ($database->isUserEmailIdExits($_POST['email'])) {
+    $GLOBALS["emailIdErrorMessage"] = "Account already exits.You can login.";
+  }
+  else {
+    $cookiePolicy = 0;
+    if ($_POST['cookiePolicy'] == "accept") {
+      global $cookiePolicy;
+      $cookiePolicy = 1;
+    }
+    $database->registerUser($_POST['firstName'], $_POST['lastName'], $_POST['radio'],
```

Comment:
How are we handling cookie policy for anonymous users?

Reply:
Sir, whenever an anonymous user enters the website I first show a popup mentioning all the cookie policies and there I show accept and decline options. If the user prefers to accept the cookie I store it as a cookie on the client side.